usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -3325,10 +3325,6 @@ static CK_RV import_DSA_key(STDLL_TokData_t * tokdata, SESSION * sess,
          * Builds the DER encoding (ansi_x962) SPKI.
          * (get the length first)
          */
-        rc = ber_encode_DSAPublicKey(TRUE, &data, &data_len,
```

Comment:
Commit title: 'Fix memory leak on DES key creation' should be '...**DSA** key creation' I guess.

Reply:
Yes.  Well, two out of tree letters correct isn't that bad...  Fixed.

Reply:
Also removed the `(get the length first)` from this comment.